mucgpt-frontend/src/components/ChatsettingsDrawer/ChatsettingsDrawer.module.css

```
@@ -22,6 +22,43 @@
+
+@media (max-width: 860px) {
+    .compactRadioGroup {
+        grid-template-columns: 1fr;
+    }
+
+    .compactRadioGroup [role="radio"] {
+        min-height: auto;
+    }
+}
+
+.compactRadioGroup [role="radio"] :global(.fui-Button__content) {
```

Stylelint (17.14.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)